blog/2022-10-07-release-1.3.md

```diff
@@ -0,0 +1,255 @@
+    ...
+  patchPodMetadata:
+  - annotations:
+      oom-score: '{"log-agent": 1}'
+      custom.example.com/sidecar-configuration: '{"command": "/home/admin/bin/start.sh", "log-level": "3"}'
+    patchPolicy: MergePatchJson
+    - annotations:
+      apps.kruise.io/container-launch-priority: Ordered
+    patchPolicy: Overwrite | Retain
+```
+patchPolicy is the injected policy, as follows:
+- **Retain:** By default, if annotation[key]=value exists in the Pod, the original value of the Pod will be retained. Inject annotations[key]=value2 only if annotation[key] does not exist in the Pod.
+- **Overwrite:** Corresponding to Retain, when annotation[key]=value exists in the Pod, it will be overwritten value2.
+- **MergePatchJson:** Corresponding to Overwrite, the annotations value is a json string. If the annotations[key] does not exist in the Pod, it will be injected directly. If it exists, do a json value merge.
+For example: annotations[oom-score]='{"main": 2}' exists in the Pod, after injection, the value json is merged into annotations[oom-score]='{"log-agent": 1, "main": 2}'.
+
+**Note:** When the patchPolicy is Overwrite and MergePatchJson, the annotations can be updated synchronously when the SidecarSet in-place update the Sidecar Container.
+However, if only the annotations are modified, it will not take effect. It must be in-place update together with the sidecar container image.
+When patchPolicy is Retain, the annotations will not be updated when the SidecarSet in-place update the Sidecar Container.
+
+According to the above configuration, when the sidecarSet is injected into the sidecar container, it will inject Pod annotations synchronously, as follows:
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  annotations:
+    apps.kruise.io/container-launch-priority: Ordered
+    oom-score: '{"log-agent": 1, "main": 2}'
+    custom.example.com/sidecar-configuration: '{"command": "/home/admin/bin/start.sh", "log-level": "3"}'
+name: test-pod
+spec:
+  containers:
+    ...
+```
+
+### 5. Advanced DaemonSet support pre-downloading image for update
+
+If you have enabled the `PreDownloadImageForDaemonSetUpdate` feature-gate,
+DaemonSet controller will automatically pre-download the image you want to update to the nodes of all old Pods.
+It is quite useful to accelerate the progress of applications upgrade.
+
+The parallelism of each new image pre-downloading by DaemonSet is `1`, which means the image is downloaded on nodes one by one.
+You can change the parallelism using `apps.kruise.io/image-predownload-parallelism` annotation on DaemonSet according to the capability of image registry,
+for registries with more bandwidth and P2P image downloading ability, a larger parallelism can speed up the pre-download process.
+
+You can also use `apps.kruise.io/image-predownload-min-updated-ready-pods` to make sure the new image starting pre-download after a few new Pods have been updated ready. Its value can be absolute number or percentage.
```

Comment:
currently the DaemonSet support of apps.kruise.io/image-predownload-min-updated-ready-pods is still problematic , plz add a notice here